src/nifreeze/data/base.py

```diff
@@ -100,6 +100,7 @@ def __len__(self) -> int:
         return self.dataobj.shape[-1]
 
     def _getextra(self, idx: int | slice | tuple | np.ndarray) -> tuple[Unpack[Ts]]:
+        _ = idx  # Avoid unused parameter warning
         return ()  # type: ignore[return-value]
```

Comment:
```suggestion
        return None  # type: ignore[return-value]
```

Reply:
Same as https://github.com/nipreps/nifreeze/pull/291#discussion_r2464095197. Thanks for the review !